onelogin_aws_cli/argparse.py

```diff
@@ -34,7 +34,7 @@ def __init__(self):
         )
 
         self.add_argument(
-            '-d', '--duration-seconds', type=int, default=3600,
+            '-d', '--duration-seconds', type=int,
```

Comment:
What are the pros/cons of setting defaults via the config as you have versus just putting defaults here?

Reply:
If (as an example), a user were to have the `duration_seconds` value in the config file, and not provide it via the cli, as the cli has precedence to overwrite values in the config file, when we parse the args, this default value of `3600` will always take precedence over the values in the config file.

If the config precedence chain has the cli options higher than the config file, then defaults must be set at the bottom level of the chain (in this case, the config file).